### DISCUSSION TITLE
Nauka, materiały

### DISCUSSION THREAD

**Discussion by sirkubax** [AzureCommunityPL/group#256] [Ogólna]
Siema
Znajomy mnie pyta, z czego uczyć się Azure (j. angielski) 🙂 i w sumie nie wiem co teraz dobre na początek.
Na ile rozumiem, to narazie szuka bezpłatnych materiałów.
Coś polecacie? Co jest teraz modne 😃

**Comment by rmandziarz** [comment]
Jak za darmo i na początek to MS Learn: https://docs.microsoft.com/en-us/learn/browse/?products=azure + konto trial Azure. Na MS Learn są laby, więc jak znalazł na początek.

**Comment by PawelHaracz** [comment]
Kanał Adama https://marczak.io/

**Comment by siudeks** [comment]
[MS Learn](https://docs.microsoft.com/pl-pl/learn/)
[Azure architektura](https://docs.microsoft.com/pl-pl/azure/architecture/browse/)
[Azure Citadel](https://azurecitadel.com/)
[Terraform Azure](https://learn.hashicorp.com/collections/terraform/azure-get-started)
[Azure DevOps labs](https://www.azuredevopslabs.com/)
[Azure serverless community library](https://www.serverlesslibrary.net/)
[Azure code samples](https://docs.microsoft.com/en-us/samples/browse/?products=azure)
[Azure AI](https://www.microsoft.com/en-us/ai/ai-school)

**Comment by krukowskid** [comment]
Bardzo lubię słuchać tego gościa: https://www.youtube.com/user/NTFAQGuy John Savill
Poza tym MS Learn ma moim zdaniem bardzo dobry kontent i fajnie pogrupowany.

**Comment by miqm** [comment]
Ja z kolei polecę jako wstępniaki do różnych technologii serię [Azure Friday](https://azure.microsoft.com/en-us/resources/videos/azure-friday/) Bardzo fajnie się tego słucha i daje dość dobry pogląd na omawiany temat.
